src/kubernetes.cc

```diff
@@ -1049,9 +1049,13 @@ void KubernetesReader::PodCallback(
     MetadataUpdater::UpdateCallback callback,
     const json::Object* pod, Timestamp collected_at, bool is_deleted) const
     throw(json::Exception) {
```

Comment:
update the method signature - since the exception is getting got and logged, the throw(..) can be dropped.